client/src/modules-data.json

```diff
@@ -0,0 +1,176 @@
+{
+  "modules": [
+    {
+      "id": 1,
+      "module_name": "Html-Css",
+      "topics": [
+        {
+          "topic_id": 1,
+          "topicName": "Fundamentals • Semantics",
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 2,
+          "topicName": "Forms • Structuring Data",
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 3,
+          "topicName": "Layout • Flex box • Grid",
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 3,
+          "topicName": "Layout • Flex box • Grid",
+          "topicRef": "www.HTML-CSS.com"
+        }
+      ]
+    },
+    {
+      "id": 2,
+      "module_name": "JavaScript_1",
```

Comment:
I think maybe a name exactly like in the CYF syllabus will be good to have, like "JavaScript_Core_1', following with the topic name "Hello World (of JS): variables, strings, types, numbers, statements, expressions, functions," etc.

Reply:
I have fixed the topic name.